apps/backoffice-v2/src/pages/Entity/components/Case/Case.Documents.tsx

```diff
@@ -56,7 +58,9 @@ export const Documents: FunctionComponent<IDocumentsProps> = ({
         <div
           className={ctw(
             `
-            d-full relative flex justify-center rounded-md`,
+            d-full relative flex rounded-md`,
+            { 'justify-center': isPage },
+            { 'justify-end': !isPage },
```

Comment:
Instead of passing isPage and handling these classes at this nesting level it makes a lot of sense to be handled by a container or to allow passing className.

Reply:
Not relevant after latest changes.